review/npm.md

```diff
@@ -119,64 +122,64 @@ typically contains a `main`, `exports`, `browser`, `module`, and/or `types` entr
 
 - **Standalone CLIs**: These are projects published on the npm registry
 and consumed by end-users in the form of locally installed programs that
-are **always** run stand alone via `npx` or via a global install.
+are **always** run stand-alone via `npx` or via a global install.
 An example would be [clipboard-cli](https://github.com/sindresorhus/clipboard-cli).
 (Their manifest file contains a `bin` entry).
 
 - **Application projects**: These are projects that teams collaborate on in
-development and deploy, such as web sites and/or web applications. 
+development and deployment, such as websites and/or web applications. 
 An example would be a React web app for a company's user-facing SaaS.
 (Their manifest file typically contains `"private": true`).
 
 ### Reproducible installation
 
-A reproducible installation is one that guarantees the exact same copy the
+A reproducible installation guarantees an exact, identical copy the
 dependencies are used each time the package is installed. This has various
 benefits, including:
 
 - Ensuring the dependencies installed are the ones declared and reviewed via
   pull requests.
 
-- Helping quickly identify possible compromises of your infrastructure if one
+- Helping specific possible compromises of your infrastructure if one
```

Comment:
verb is missing?

Reply:
`quickly` can be subjective and is used as an adverb. I removed it because it made little difference to the message's meaning, and generally, it's best to avoid subjective contexts; however, I'm open to ideas if we think it's best in there.

Reply:
you removed `identify` though, which makes the sentence incorrect.

I think helping *quickly* identify, versus merely identifying, is actually important to the message's meaning.

Reply:
> you removed identify though, which makes the sentence incorrect.

Yup, very good, thank you.

> I think helping quickly identify, versus merely identifying, is actually important to the message's meaning.

Fair point. From a grammatical perspective, I still believe the use of the word `quickly` in this instance could still be considered [a subjective squinting modifier](https://www.thoughtco.com/squinting-modifier-grammar-1692130). 

Either way, it's been reverted to what it was initially, so I believe that resolves this matter.

Reply:
I don't agree - "quickly" here doesn't modify "helping", it only modifies "identify", and there's no rearrangement I can think of that would work except putting "quickly" at the end, which would diminish its emphasis and thus the point of including the adverb.

Reply:
Hey @ljharb, sorry I didn't get the opportunity to respond earlier. I agree with your presented context; it makes solid sense. I'm simply stating, though, from a technical grammar point-of-view, that the squinting modifier isn't optimum. That said, I'm okay with leaving it as-is (with my changes retracted), given the audience and context of the document.

FWIW, not to be argumentative, I did double-check this sentence though on [Hemingway](https://hemingwayapp.com/) and [Grammarly](https://app.grammarly.com/); both software picked it up as a squinting modifier and recommended removing it.

Reply:
Fair enough :-) i still think it conveys meaning tho.